### ISSUE TITLE
Compare val set and train set for epochs: Overfitting? Pattern?

### ISSUE THREAD

**Issue by ckaarle** [ckaarle/ma#44]
yes, patterns also show up in validation set.
